includes/Modules/Analytics.php

```diff
@@ -562,6 +563,22 @@ protected function create_data_request( $method, $datapoint, array $data = array
 						}
 						return false;
 					};
+				case 'tag-permission':
+					return function() use ( $data ) {
+						if ( ! isset( $data['tag'] ) ) {
+							/* translators: %s: Missing parameter name */
+							return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'tag' ), array( 'status' => 400 ) );
+						}
+						$accounts               = $this->get_data( 'get-accounts' );
+						$has_access_to_property = $this->has_access_to_property( $data['tag'], $accounts['accounts'] );
+
+						if ( empty( $has_access_to_property ) ) {
+							/* translators: %s: Property id of the existing tag */
+							return new WP_Error( 'google_analytics_existing_tag_permission', sprintf( __( 'We\'ve detected there\'s already an existing Analytics tag on your site (ID %s), but your account doesn\'t seem to have access to this Analytics property. You can either remove the existing tag and connect to a different account, or request access to this property from your team.', 'google-site-kit' ), $data['tag'] ), array( 'status' => 500 ) );
```

Comment:
A `5xx` response here seems like the wrong status code to send as there is no internal error, exception caught, etc. Is there a reason this is necessary? If not, I think something like a `403` would be more appropriate.

Also, please split this onto multiple lines as well as it is quite long, thank you!